text/0061-error-format-and-definitions.md

```diff
@@ -1350,6 +1350,30 @@ HTTP Code: `403 Forbidden`
 
 ---
 
+## missing_master_key
+
+`Synchronous`
+
+### Context
+
+For some specific protected routes (i.e. `/keys`) where the master key is mandatory to access it, if the users do not pass any `Authorization` header, this error occurs instead of `missing_authorization_header`.
+If an `Authorization` is passed but with the wrong master key value, `invalid_api_key` is still returned.
+
+### Error Definition
+
+HTTP Code: `401 Forbidden`
+
+```json
+{
+    "message": "The master key is needed to access this route.",
```

Comment:
```suggestion
    "message": "Melisearch is running without a master key. To access this API endpoint, you must have set a master key at launch.",
```

This is a suggestion; feel free to make it clearer. We could also use the link to the doc in the message https://docs.meilisearch.com/learn/security/master_api_keys.html#protecting-a-meilisearch-instance.